kernel/rros/clock.rs

```diff
@@ -725,11 +681,10 @@ fn init_clock(clock: *mut RrosClock, master: *mut RrosClock) -> Result<usize> {
     Ok(0)
 }
 
-//初始化时钟slave
 fn rros_init_slave_clock(clock: &mut RrosClock, master: &mut RrosClock) -> Result<usize> {
     premmpt::running_inband()?;
 
-    //这里为什么会报错，timer就可以跑？为什么卧槽
+    // Why is an error reported here? Timer can run? Why the hell?
```

Comment:
Modify the statement here. If it has been completed, delete the comment. Otherwise add a todo here.

Reply:
OK.